src/demo-app/a11y/button/button-a11y.html

```diff
@@ -1,31 +1,52 @@
 <div class="demo-button">
   <section>
     <h2>Button elements</h2>
-    <button md-button>Check</button>
-    <button md-raised-button>raised</button>
-    <button md-fab aria-label="Activate floating action style button">
-      <md-icon class="md-24">check</md-icon>
+    <p>Click on the buttons to increase the button counter.</p>
+    <p>Current number of clicks: {{counter}}</p>
+    <button md-button (click)="increase()">Check</button>
```

Comment:
What prompted you to expand these with the additional functionality? Something from the guidelines?